tests/graph/test_planar.py

```diff
@@ -0,0 +1,283 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import itertools
+import retworkx
+
+
+class TestPlanarGraph(unittest.TestCase):
```

Comment:
Is there a reason tests are duplicated in rust and python? Is that the standard approach for `retworkx-core` code?

Reply:
It's not standard but it's something that we'd like to do in the future (see #587) since `retworkx-core` is a separate Rust crate and it deserves its own testing. Here I originally wrote the tests in rust since it's faster to built and run the rust tests and it saves some development time.